coverage/.last_run.json

```diff
@@ -1,5 +0,0 @@
-{
-  "result": {
-    "covered_percent": 34.94
```

Comment:
I don't think you want to commit this bit

Reply:
OK whoops